.changeset/smart-forks-pay.md

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
Duplicate changeset?

Reply:
Also should the new package technically be a major bump?